packages/core/components/ComponentList/index.tsx

```diff
@@ -17,51 +31,67 @@ export const ComponentList = ({ config }: { config: Config }) => {
           ref={provided.innerRef}
           className={getClassName()}
         >
-          {Object.keys(config.components).map((componentKey, i) => {
-            const componentConfig: ComponentConfig = config[componentKey];
+          {Object.entries(groupedComponents)
+            .sort()
+            .map(([tag, components]) => {
+              return (
+                <>
+                  {Object.keys(groupedComponents).length > 1 ? (
+                    <Heading rank={3} size={"xs"}>
+                      {tag}
+                    </Heading>
+                  ) : null}
```

Comment:
I'm not totally sure that using Headings like this is the way to go for the UI here.

It also exacerbates some existing visual issues when cancelling a drag


https://github.com/measuredco/puck/assets/985961/5590e311-029e-42e5-a6db-4c42d2c7e627



I wonder if we should render multiple